### PR TITLE
implement the booking controller

### PR DESCRIPTION
**Pull request by Thibault-Sordet**

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     @bookings = Booking.joins(:star).where("stars.user_id = #{current_user.id}")
+  end
+
+  def show
+    @review = Review.new
+  end
+
+  def destroy
+    @booking.destroy
+    redirect_to star_path(@star), notice: 'Booking was successfully destroy'
```
Comment (NicolasNeiman) [inline app/controllers/bookings_controller.rb:33]: Confirm ?

File: app/views/bookings/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<br>
+<br>
+<br>
+<br>
+<br>
+<%= simple_form_for [ @booking, @review ] do |f| %>
+  <%= f.input :rating, :required => false %>
+  <%= f.input :comment, :required => false %>
+  <%= f.submit "Add a review" %>
+<% end %>
```
Comment (NicolasNeiman) [inline app/views/bookings/show.html.erb]: Do we need a booking page if we have a my_bookings and a my_listings ?

**Review by NicolasNeiman — CHANGES_REQUESTED** [review]

File: app/views/bookings/my_booking.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+<ol>
+  <% @bookings.each do |booking| %>
+  <li> <%= booking.star.name %> / </li>
```
Comment (NicolasNeiman) [inline app/views/bookings/my_booking.html.erb]: add delete button ?

Comment (Thibault-Sordet) [reply app/views/bookings/my_booking.html.erb]: done

File: app/views/bookings/show.html.erb (hunk 2 above)
Comment (Thibault-Sordet) [reply app/views/bookings/show.html.erb]: yes because it's a show page and not an index page

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (Thibault-Sordet) [reply app/controllers/bookings_controller.rb:33]: done in the delete btn

**Review by NicolasNeiman — APPROVED** [review]

**Pull request merged**